### PR TITLE
Con 3583 view review cohort

### PR DESCRIPTION
**Pull request by hkf-tech**

### CODE REVIEW

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/CohortControllerTests/WhenGettingDetails.cs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        [Test]
+        public async Task ThenViewModelShouldBeMappedFromRequest()
+        {
+            await _fixture.GetDetails();
+            _fixture.VerifyViewModelIsMappedFromRequest();
+        }
+
+        [TestCase(Party.Employer)]
+        [TestCase(Party.TransferSender)]
+        public async Task ThenViewModelIsReadOnlyIfCohortIsNotWithEmployer(Party withParty)
```
Comment (cofaulco) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/CohortControllerTests/WhenGettingDetails.cs:37]: Should this be "Is With Employer" or "Is not with provider"?

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs [hunk 2]
```diff
@@ -164,7 +164,56 @@ public async Task<IActionResult> Delete([FromServices] IAuthenticationService au
...
+                    {
+                        return RedirectToAction("Cohorts", new { viewModel.ProviderId });
+                    }
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(viewModel.Selection));
+            }
+        }
+
+        [HttpGet]
+        [Route("{cohortReference}/Acknowledgement")]
```
Comment (cofaulco) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs:211]: ```[DasAuthorize(CommitmentOperation.AccessCohort)]``` ??

**Review by cofaulco — COMMENTED** [review]
1 minor cosmetic thing

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/CohortControllerTests/WhenGettingDetails.cs (hunk 1 above)
Comment (hkf-tech) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/CohortControllerTests/WhenGettingDetails.cs:37]: Yes, it should be

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs (hunk 2 above)
Comment (hkf-tech) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs:211]: I merged these changes from 3737 branch.

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/AcknowledgementRequestViewModelMapper.cs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            {
+                result.PageTitle = "Cohort approved";
+            }
+            else
+            {
+                result.PageTitle = source.SaveStatus == SaveStatus.ApproveAndSend
+                    ? "Cohort approved and sent to employer"
+                    : "Cohort sent to employer for review";
+            }
+
```
[the remark this answers is not in the document]
Comment (hkf-tech) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/AcknowledgementRequestViewModelMapper.cs:78]: fixed

**Review by cofaulco — APPROVED** [review]

**Pull request merged**